tests/Export/ProductServiceTest.php

```diff
@@ -421,4 +423,224 @@ public function testProductIsNotFoundIfMainVariantIsNotAvailableForSalesChannel(
 
         $this->assertEmpty($result->getElements());
     }
+
+    public function mainVariantDefaultConfigProvider(): array
+    {
+        $parentId = Uuid::randomHex();
+
+        return [
+            'export shopware default' => [
+                'config' => 'default',
+                'parentId' => $parentId,
+                'parentPrice' => 15,
+                'cheapest' => 2
+            ],
+            'export main parent' => [
+                'config' => 'parent',
+                'parentId' => $parentId,
+                'parentPrice' => 15,
+                'cheapest' => 2
```

Comment:
Please always make sure that the data provider names match the method arguments, otherwise this gets a bit confisuing. `cheapest` would be `cheapestPrice`.